### ISSUE TITLE
Request: libwebp.js

### ISSUE THREAD

**Issue by ohenepee** [SaschaNaz/libflif.js#18]
Hello Saschanaz... thank you for this awesome lib... I've used it to lossy compressed images and compare them to WebP. And it works perfectly, I just wanted to ask if you could do the same for WebP format (an in-browser encoder) they seem to already have a decoder in their repo... Perhaps you could also do one for mozjpeg and bundle all these libs into one awesomecore library `WebImageCodecs`. Would be glad to see the WebP wasm/asm.js version. Thanks

**Comment by saschanaz** [comment]
It seems there already is one: http://libwebpjs.appspot.com/ which supports encoding.

**Comment by ohenepee** [comment]
Yes... thanks for even looking the idea up... really appreciate it. I had already checked that out. A few things I noticed about it are:
- No WASM or ASM.JS implementation (it is 100% JS, :-1: in performance)
- No Source Code ( Check his [repos](https://github.com/dominikhlbg) )
- He hand ports most of his ported projects, which means they don't really follow upstream releases

**Comment by saschanaz** [comment]
Hmm, so it seems worth trying, although currently I don't have enough interest to work on it...

**Comment by ohenepee** [comment]
I perfectly understand... probably at ur free time... will keep checking ur repos to see when its done though :)
